src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -241,6 +241,11 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
     }
 
     this._setTemp = this._getSetTemp(this.hass!.states[this._config!.entity]);
+    this._longName =
```

Comment:
I was more thinking of making `name` and const in render, as we now have to computeStateName and check if `config` has name twice (line 167). No need to make it a property.